frontend/src/pages/Chat/components/Room/Room.tsx

```diff
@@ -1,26 +1,35 @@
+import { useLocalStorage } from "usehooks-ts";
 import Initials from "../../../../components/Initials/Initials";
 import { ChatRoomDto } from "../../../../types/room";
+import { isCurrentUserParticipantOne } from "../../../../utils/utils";
 import styles from "./styles.module.scss";
+import { AccountDto } from "../../../../types/account";
 
 type Props = {
   roomData: ChatRoomDto;
   onRoomClick: () => void;
 };
 
 const Room = ({ roomData, onRoomClick }: Props) => {
+  const [accountDetails] = useLocalStorage<AccountDto | null>("account", null);
   const timeToDisplay = new Date(roomData.UpdatedAt).toLocaleTimeString();
   const maxTextLength = 32;
   const messageContentParsed =
     roomData.LastMessage.length > maxTextLength
       ? `${roomData.LastMessage.slice(0, maxTextLength)}...`
       : roomData.LastMessage;
+  if (!accountDetails) return null;
+  const recipient =
```

Comment:
I would extract this function as a helper into utils  and not just the iscurrentuser part